libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/MapboxNavigationViewModel.kt

```diff
@@ -33,7 +33,7 @@ import kotlinx.coroutines.flow.MutableSharedFlow
 import kotlinx.coroutines.launch
 
 @OptIn(ExperimentalPreviewMapboxNavigationAPI::class)
-internal class MapboxNavigationViewModel : ViewModel() {
```

Comment:
Why did we remove `internal`? We should not allow access to any View model to outside developers